nixGL.nix

```
@@ -81,7 +81,7 @@ let
           else
             builtins.fetchurl url;
           useGLVND = true;
-          nativeBuildInputs = oldAttrs.buildInputs ++ [zstd];
+          nativeBuildInputs = oldAttrs.buildInputs or [] ++ [zstd];
```

```suggestion
          nativeBuildInputs = oldAttrs.nativeBuildInputs or [] ++ [zstd];
```

Override the correct attribute; as it currently exists this breaks the nixpkgs driver builder since it discards the `libarchive` dependency.

---

I've done so now, trusting your word on this :) but I myself don't know enough about it. I've quickly tried it out and it seems to build and work just fine, but in principle this PR now fixes two different things at once.